spec/models/mixins/authentication_mixin_spec.rb

```diff
@@ -266,40 +266,32 @@ def self.name; "TestClass"; end
       end
     end
 
-    context ".validate_credentials_task" do
-      let(:args) { %w(userid password foo) }
+    context ".verify_credentials_task" do
+      let(:clazz) { ManageIQ::Providers::Redhat::InfraManager }
```

Comment:
I think "klass" is the typical substitute, but yours is more...jazzy I guess. ;)